app/src/main/java/com/example/bookwormadventuresdeluxe2/NotificationListAdapter.java

```diff
@@ -0,0 +1,77 @@
+package com.example.bookwormadventuresdeluxe2;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.constraintlayout.widget.ConstraintLayout;
+import androidx.recyclerview.widget.RecyclerView;
+
+import java.util.ArrayList;
+
+public class NotificationListAdapter extends RecyclerView.Adapter<NotificationListAdapter.NotificationListViewHolder>
```

Comment:
This looks very similar to the BookListAdapter, except for the layout. Is there any way to try and reuse BookListAdapter for the notifications page?

Reply:
I reused the book_item xml but I think trying to reuse the adapter would require overloading basically everything

Reply:
Ok. I ended up overhauling the BookListAdapter to extend FirestoreRecyclerAdapter anyways so I think this is okay for now.